docs/modules/ROOT/pages/reference/arch-control-plane.adoc

```diff
@@ -0,0 +1,397 @@
+It provides the mechanisms for the multi-tenant, multi-service-provider self-service portal.
+
+CSP Control Plane (CSPCP)::
+This runs Crossplane and the AppCat Control Plane which manages all the service instances on worker clusters.
+Each zone of a CSP runs at least one CSPCP.
+
+Worker Cluster::
+Clusters running the actual workload, managed by the CSPCP.
+Each zone of a CSP runs at least one Worker Cluster.
+Services are directly exposed to the end-user.
+
+Service::
+A managed service which can be instantiated (ordered) and parametrized according to the specification.
+
+Plan::
+A pre-defined set of parameters for a service.
+
+Crossplane specific terms::
+* Compositions - A template to define how to create resources.
+* Composite Resource Definition (`XRD`) - A custom API specification.
+* Composite Resource (`XR`) - Created by using the custom API defined in a Composite Resource Definition. XRs use the Composition template to create new managed resources.
+* Claims (`XRC`) - Like a Composite Resource, but with namespace scoping.
+* Managed Resource (`MR`) - Represents an external service in a Provider.
+* Providers enable Crossplane to provision infrastructure on an external service. Providers create new Kubernetes APIs and map them to external APIs.
+
+== Organization Enhancements
+
+=== Organization Origin
+
+This enhances https://kb.vshn.ch/appuio-cloud/references/architecture/control-api-org.html[APPUiO Control API: Organization^] with the aspect of the origin of an organization.
+The origin specifies from where an organization originates, either from VSHN, a CSP like Exoscale or cloudscale or any other entity taking part.
+
+This origin is used in further processes, like filtering of resources or access control.
+
+It's part of the `Organization` resource, in `.spec.originRef` which references a `OrganizationOriginConfig` (see <<Organization Origin Configuration>>.
+
+.Example Organization
+[source,yaml]
+----
+apiVersion: organization.appuio.io/v1
+kind: Organization
+metadata:
+  name: acme-corp 
+spec:
+  displayName: Acme Corp.
+  originRef: exoscale
+----
+
+.Example Organization Namespace
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: acme-corp
+  labels:
+    appuio.io/resource.type: organization
+    vshn.io/organization-ref-name: exoscale <1>
+  annotations:
+    organization.appuio.io/display-name: Acme Corp.
+----
+<1> The value of `spec.originRef.name` from the `Organization` resource
+
+=== Organization Origin Configuration
+
+An organization origin is configured a resource kind called `OrganizationOriginConfig`.
+This resource is referenced in the `.spec.originRef` of the `Organization` resource kind and is used to parametrize further processes.
+
+[source,yaml]
+----
+apiVersion: vamp.vshn.io/v1
+kind: OrganizationOriginConfig
+metadata:
+  name: exoscale
+spec:
+  compositionSelector: <1>
+    metadata.appcat.vshn.io/serviceprovider: exoscale
```

Comment:
Maybe this should be an array of `matchlabels`? Each selector will then be `OR`'d.

For example if an org has access to exoscale and cloudscale it would look something like this:
```
compositionSelector:
    - matchLabels:
         metadata.appcat.vshn.io/serviceprovider: exoscale
    - matchLabels:
         metadata.appcat.vshn.io/serviceprovider: cloudscale
```

This will then show both providers.